workflow/scripts/postprocess_logcount.py

```
@@ -1,3 +1,6 @@
+import sys
+sys.stderr = open(snakemake.log[0], "w")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)